CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Don't fail to start the dogu if any of the Nexus health checks fails; #51
```

Comment:
We should be clear about changes to limits we imposed earlier. The CPU error f. i. was a hard No. Since that changes we could mention this.

```suggestion
- Don't fail to start the dogu if any of the Nexus health checks fails; #51
   - less than 4 processors do no longer raise an error and will not lead to a Nexus reboot
   - as [Nexus still requires 4 processors](https://issues.sonatype.org/browse/NEXUS-19839) a warning will be logged instead
```